docker/templates/DBDefs.pm.ctmpl

```diff
@@ -188,7 +188,7 @@ sub GIT_SHA { {{executeTemplate "QUOTED_STRING" (env "GIT_SHA")}} }
 {{executeTemplate "BOOLEAN_DEF" "DEVELOPMENT_SERVER"}}
 {{executeTemplate "BOOLEAN_DEF" "FORK_RENDERER"}}
 {{executeTemplate "BOOLEAN_DEF" "IS_BETA"}}
-{{executeTemplate "BOOLEAN_DEF" "LUCENE_X_ACCEL_REDIRECT"}}
+{{executeTemplate "BOOLEAN_DEF" "SEARCH_X_ACCEL_REDIRECT"}}
```

Comment:
I think we need compatibility for old configs here: `LUCENE_X_ACCEL_REDIRECT` or `SEARCH_X_ACCEL_REDIRECT`
@mwiencek : ^^ ?

Reply:
It might not be necessary, since the .ctmpl is only used by our production servers and we'll have to fix our consul config anyway to add `SEARCH_ENGINE` etc. But we'd have to make sure to update the config files correctly (leaving the old keys in place until after the new code is deployed).